Formula/ffmpeg.rb

```diff
@@ -3,6 +3,7 @@ class Ffmpeg < Formula
   homepage "https://ffmpeg.org/"
   url "https://ffmpeg.org/releases/ffmpeg-4.1.4.tar.xz"
   sha256 "f1f049a82fcfbf156564e73a3935d7e750891fab2abf302e735104fd4050a7e1"
+  revision 1
```

Comment:
Please include each revision bump as a separate commit in the same PR, with message (for example) `ffmpeg: revision for x265`

Reply:
Done. I'll make sure this gets a mention in the docs.